container/functional/keylime_verifier_registrar_container-basic-attestation/main.fmf

```diff
@@ -0,0 +1,32 @@
+summary: Tests basic keylime attestation scenario for keylime agent,verifier and registrar containers
+description: |
+    Running agents,verifier and registrar in containers.
+    Only tenant tool is run on localhost.
+    Every container uses certificates generated by keylime verifier.
+    Build container image for agent, verifier and registrar.
+    Starts verifier, registrar in container.
+    Setup agent conf for agent.
+    Run container with agent.
+    Register agent by verifier.
+    Verifiers that container passed with agent attestation.
+    Do changes in scripts monitored by agent and confirm that agent fail attestation accordingly.    
+contact: Patrik Koncity <pkoncity@redhat.com>
+tag:
+  - container
+component:
+  - keylime
+test: ./test.sh
+framework: beakerlib
+require:
+  - yum
+  - podman
+  - nmap
```

Comment:
We are importing `test-helpers` library directly and therefore `require` of the library itself won't have an effect.. And therefore you are adding it to a test.
I am thinking, what about installing all library requires during the library import so we won't have to add `nmap` to every test using it?